src/main/treeDataProviders/utils/analyzerUtils.ts

```diff
@@ -247,8 +251,12 @@ export class AnalyzerUtils {
             }
             let cvesToScan: Set<string> = new Set<string>();
             fileScanBundle.dataNode.issues.forEach((issue: IssueTreeNode) => {
-                if (issue instanceof CveTreeNode && !issue.parent.indirect && issue.cve?.cve) {
-                    // Only direct cve issues
+                if (!(issue instanceof CveTreeNode) || !issue.cve?.cve) {
+                    return;
+                }
+                // For Python projects, all CVEs should be included.
```

Comment:
```suggestion
                // For Python projects, all CVEs should be included because in some cases it is impossible to determine whether a dependency is direct.
```